it/v1/endpoints/RetrievePeriodicObligationsControllerISpec.scala

```diff
@@ -314,6 +318,85 @@ class RetrievePeriodicObligationsControllerISpec extends IntegrationBaseSpec {
         |""".stripMargin
     )
 
+    val dynamicObligationsResponseBody: JsValue = Json.parse(s"""
+                                                                |{
```

Comment:
Can the indentation here be brought in line with the other jsons in the file? (i.e. way over to the left)